mlir/test/Conversion/GPUToSPIRV/printf.mlir

```diff
@@ -0,0 +1,71 @@
+// RUN: mlir-opt -allow-unregistered-dialect -split-input-file  -convert-gpu-to-spirv -verify-diagnostics %s -o - | FileCheck %s
```

Comment:
Do we need the `-o o`?

Reply:
Not really, was just trying to follow test command lines from other tests, I can remove it.